index.html

```diff
@@ -198,6 +198,55 @@ <h2>Skills</h2>
           <div class="workicon3"></div>
         </div>
       </section>
+      <section id="form">
+          <nav>
+            <div class="menu">
+              <div class="shape">
+                <div id="triangle"></div>
+                <div id="diamond"></div>
+              </div>
+            </div>
+          </nav>
+          <div class="form-title">
+          <h1 class="title">Get started</h1>
+          <p class="paragraph">
+            Hello! I am a software developer! I can help you build a product,
+            feature or website. Fill this form and lets work.
+          </p>
+          <form action="/my-handling-form-page" method="post">
```

Comment:
- [ ] The `action` attribute of the form is expected to collect data by using [Formspree service](https://formspree.io/html/). Create an account in there and you would be given an `id` that would be used for the form on your project
It would look something like this. :point_down: 
```
 <form  action="https://formspree.io/f/mwkzgobg"  method="POST">
```
. Kindly fix that.